Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+function CardContent() {
+  return (
+    <div className="cardContent">
+      <h1 className="getStarted">Get started with React</h1>
+      <p className="cardSummary">
```

Comment:
at this point, you don't want anything hard coded in your functional child components. You basically want to pass it all as props to keep it reusable.